internal/himama/client.go

```diff
@@ -157,7 +157,7 @@ func getLoginForm(client *http.Client) (csrfToken []byte, err error) {
 	}
 
 	// TODO: Using regex to parse HTML? Why not.
-	re := regexp.MustCompile(`<meta name=csrf-token content=([a-zA-Z0-9_\/+\-]+=*) />`)
+	re := regexp.MustCompile(`<meta name=csrf-token content=([a-zA-Z0-9\/+_-]+) *\/>`)
```

Comment:
A simpler approach would be just to capture what's in between the double quotes after `content=`
```suggestion
	re := regexp.MustCompile(`<meta name=csrf-token content="([^"]*)"`)
```